net/amneziawg-tools/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2016-2019 Jason A. Donenfeld <Jason@zx2c4.com>
+# Copyright (C) 2016 Baptiste Jonglez <openwrt@bitsofnetworks.org>
+# Copyright (C) 2016-2017 Dan Luedtke <mail@danrl.com>
```

Comment:
This package was adopted from some other package, here? Would be good to mention that. Otherwise someone, who does not know it, they will write them.